ddtrace/contrib/tornado/__init__.py

```diff
@@ -0,0 +1,22 @@
+"""
+TODO: how to use Tornado instrumentation
+"""
+from ..util import require_modules
+
+
+required_modules = ['tornado']
+
+with require_modules(required_modules) as missing_modules:
+    if not missing_modules:
+        from .tracer import TornadoTracer
+        from .middlewares import TraceMiddleware
+        from .stack_context import ContextManager
+
+        # a global Tornado tracer instance
+        tracer = TornadoTracer()
```

Comment:
according to a proposal, this code will be removed in favor of a way to set the default tracer.